### DISCUSSION TITLE
Change the color sliders for a palette

### DISCUSSION THREAD

**Discussion by NatoBoram** [Docile-Alligator/Infinity-For-Reddit#923] [Ideas]
In the theme customization, [it's pretty hard to get what you want since you have to edit about 6 settings to change a single colour](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/233), and this is exacerbated by the color sliders.

![Screenshot_20220302-095139](https://user-images.githubusercontent.com/10495562/156385885-1a89e225-0569-4fc0-adfa-f7819e71757f.png)

In other mobile apps, it's possible to make a selection instead of fiddling with unintuitive sliders.

![Screenshot_20220302-095426](https://user-images.githubusercontent.com/10495562/156386115-8e7c9592-b4c5-4c48-9b9d-171eecc14ba4.png)

Having a palette available plus the hex code would make both very precise edition and general picks available, satisfying both maniacs and people who just don't want to use pink but without having to select an exact amount of red/green/blue.

Material Design has color palettes readily available: https://material.io/design/color/the-color-system.html#tools-for-picking-colors

If Material Design isn't your cup of tea, there's Tailwind with color palettes a bit more neutral: https://tailwindcss.com/docs/customizing-colors

**Comment by glibg10b** [comment]
Or just use HSL sliders, which are much more intuitive than RGB sliders

**Comment by Docile-Alligator** [comment]
Thank you. I will think about this.
